### ISSUE TITLE
docs UI Kit: Move Block Spacing to TypographyPage

### ISSUE THREAD

**Issue by nkuehn** [commercetools/commercetools-docs-kit#451]
The Information boxes and Subtitle Fragments can be used with plain-text content or by rendering the the content with the full MDX content renderer if an extra blank line is included before and after the content. 

This is the built-in behavior of MDX and is practical, also because custom rendering with the fragment renderer is only reliable for all use cases when the markdown is available programmatically - when passed from an MDX page, the string inside a tag is already trimmed and removed of all newlines when it reaches the component (HTML compatibility in JSX). 

When using MDX inside JSX Tags, we now realized the that UI kit "Markdown" components are applying sometimes large spacing that is specific to when they are used directly in the body text flow, but not when they are used inside another container. 

The UI kit "Markdown" component already does factor some of these things into the "TypopgraphyPage" Wrapper that can then be applied selectively but it's not consistent.  Some Block spacing, like the large space below a list, is directly inside the list component. 

For example, the following screenshot shows that a bullet list in an information box gets a way to large bottom spacing because that's intended in the body flow content. 

![image](https://user-images.githubusercontent.com/4006139/82461717-0617b400-9abb-11ea-8285-0b2ce3993f28.png)

MDX:
```mdx
<Info>

- Inquire at [fooBar@commercetools.com](mailto:training@commercetools.com) or use the online [request form](https://ok.commercetools.com/training-offering).
- another bULLet
- Please stay healthy

</Info>
```

**Intended behavior**  No larges space in between or below components when used inside other wrapper boxes.

**Comment by nkuehn** [comment]
Test Cases to understand the behavior of the MD(X) parser:

```mdx
<Cards>
<Card>OK: **bold** microformatting in one line</Card>

<Card>

OK: **bold** microformatting in a full block separated by blank lines

</Card>

<Card>
Valid MD(X) but does parse this line as MD(X): **don't mix the styles**

**bold** microformatting in a full block separated by blank lines

</Card>
</Cards>
```

**Comment by nkuehn** [comment]
TL;DR:   We need to rather refactor some of the UI Kit styling to not have margins and other layout / text flow preconfiguration,  these things need to be set in the `TypographyPage` component's style
